.github/workflows/ci.yaml

```diff
@@ -783,6 +783,49 @@ jobs:
           name: pytest_buckets
           path: pytest_buckets.txt
           overwrite: true
+  audit-licenses:
```

Comment:
Let's move it up so the pytest jobs are together. Either after the `mypy` job or maybe even better below the `gen-requirements-all` one.

Please also add an empty line after before the new job definition.